src/components/app/DocumentList.js

```diff
@@ -186,7 +186,9 @@ class DocumentList extends Component {
             this.setState({
                 cachedSelection: null
             }, () => {
-                dispatch(setListIncludedView());
+                if (includedView.windowType === 'pickingSlot') {
```

Comment:
but before i could integrate this task, pls check and fix following issue:

![image](https://user-images.githubusercontent.com/1244701/28363803-90f21564-6c8a-11e7-89e8-bb0b830a2d2a.png)

i.e. the quickActions endpoint is called using the previous windowId/viewId (i.e. the pickingSlot) instead of the current windowId/viewId (the HU).